### PR TITLE
Chem Plumbing Interfaces V2

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

tgui-nexts the reaction chamber because I forgot to do it, as well as updating a couple of the original interfaces with new features.
Also a slight tweak to Input's css to make the text the same color as NumberInput's, as well as the addition of a placeholder functionality

Pill Press:
image
Chem Filter:
image
Reaction Chamber:
image

## Changelog
:cl:
add: Plumbing Reaction Chamber uses tgui-next
tweak: A couple of the other plumbing uis now use the Input component of tgui-next instead of having popup windows
/:cl:

### CODE REVIEW

**Review by stylemistake — COMMENTED** [review]
I have doubts about merging this PR considering that Input is still a complete shitshow for IE8 users.

We need to either create a completely custom input, similarly to how it's done in NumberInput (directly manipulate input DOM), or just sacrifice IE8 users for this one (and create a custom input anyway, but much, much later).

An example with an uplink input:

input-ie8

**Comment by actioninja** [comment]
DNM awaiting useBackend refactor for input fixes

**Comment by actioninja** [comment]
Ready now.

File: tgui-next/packages/tgui/interfaces/ChemFilter.js [hunk 1]
```diff
@@ -35,27 +43,53 @@ export const ChemFilterPane = props => {
...
-      <Grid.Column>
-        <ChemFilterPane
-          title="Right"
-          list={right}
-          state={state} />
-      </Grid.Column>
-    </Grid>
-  );
-};
+export class ChemFilter extends Component {
```
Comment (spookydonut) [inline tgui-next/packages/tgui/interfaces/ChemFilter.js]: shouldnt this be in /components/

**Review by stylemistake — DISMISSED** [review]

**Comment by stylemistake** [comment]
I have reverted that CSS thing which colors text with `$border-color`, but added a placeholder style, and fixed one peculiar bug with "undefined" sneaking into the input. Nothing ground breaking though, so imma merge this right now.

**Pull request merged**